itanium_demangler/__init__.py

```diff
@@ -884,6 +884,19 @@ def parse(raw):
         ast = _expand_arg_packs(ast)
     return ast
 
+def is_ctor_or_dtor(raw) -> bool:
+    ast = parse(raw)
+    return ast and _is_ctor_or_dtor(ast)
+
+def _is_ctor_or_dtor(ast) -> bool:
```

Comment:
I think it's okay to export the inner `_is_ctor_or_dtor` function directly? It's nice to have a `parse` function that returns an AST and then the others operating on it.